cinema/views.py

```diff
@@ -127,6 +124,54 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
In the 'upload_image' method, you are returning a 400 BAD REQUEST status code if the serializer is invalid. It would be helpful to include more specific error messages to aid in debugging.